### PR TITLE
Add Google App Script script to send filtered opened CFPs to a chat service

### PR DESCRIPTION
**Pull request by thierrylee**
Add an utility script to display CFPs with some filters that can be used on Google Apps Script.

* [x] Filter methods
* [x] Display using Markdown
* [x] Example of sending message to a chat API
* [x] Link to a public Google App Script

[Example of a Google App Script project](https://script.google.com/d/1bt680byslxMyruogvlkCucjfgcwX_FlEjEgEpp--2HCjp45dRbKCL93v/edit?usp=sharing) (read-only)
Example output:
<details>
  <summary>Markdown style</summary>

### Ongoing CFPs from 20/5/2024 to 26/5/2024

|Name|🗓️ Dates|📍 Location|🗣️ CFP Status|
|---|---|---|---|
|[We Love Speed](https://www.welovespeed.com/)|17/9/2024|Nantes (France)|[🔴 Open until 26/5/2024](https://www.welovespeed.com/2024/cfp)|
|[Agile Tour Sophia-Antipolis 2024](https://www.telecom-valley.fr/agile-tour-sophia-2024/)|26/9/2024|Biot (France)|[🔴 Open until 31/5/2024](https://sessionize.com/agile-tour-sophia-antipolis-2024/)|
|[Agile Tour Rennes 2024](https://agiletour.agilerennes.org/)|13/11/2024 au 14/11/2024|Rennes (France)|[🔴 Open until 31/5/2024](https://sessionize.com/agile-tour-rennes-2024/)|
|[DevFest Toulouse](https://devfesttoulouse.fr/)|7/11/2024|Toulouse (France)|[🟡 Open until 2/6/2024](https://sessionize.com/devfest-toulouse/)|
|[Who Run The Tech ?](https://whorunthetech.com)|28/11/2024|Rennes (France)|[🟡 Open until 4/6/2024](https://conference-hall.io/public/event/bJDribf9r5Lf2VzBgbYQ)|
|[Agile Pays Basque](https://agile-paysbasque.fr/)|6/9/2024 au 7/9/2024|Bidart (France)|[🟡 Open until 7/6/2024](https://conference-hall.io/public/event/w59YkxhTPvNuwdaRCXKc)|
|[DevFest Nantes](https://devfest.gdgnantes.com/en/)|17/10/2024 au 18/10/2024|Nantes (France)|[🟡 Open until 9/6/2024](https://conference-hall.io/public/event/AJYPylkzTDd6j3O6ADZn)|
|[Cloud Nord](https://www.cloudnord.fr/)|10/10/2024|Lille (France)|[🟡 Open until 15/6/2024](https://conference-hall.io/public/event/WhX2AfC4tuVw5Ukdd5FW)|
|[Agile en Seine 2024](https://www.agileenseine.com/)|17/9/2024 au 18/9/2024|Issy-les-Moulineaux (France)|[🟡 Open until 16/6/2024](https://sessionize.com/agile-en-seine-2024/)|
|[JUG Summer Camp](https://www.jugsummercamp.org)|6/9/2024|La Rochelle (France)|[🟡 Open until 21/6/2024](https://conference-hall.io/public/event/p1eSeAZlUj8gc8DxE8vX)|
|[Agile Tour Bordeaux 2024](https://agiletourbordeaux.fr/)|30/10/2024 au 31/10/2024|Bordeaux (France)|[🟢 Open until 30/6/2024](https://sessionize.com/agile-tour-bordeaux-2024/)|
|[Agile Tour Nantais 2024](https://agilenantes.org/agile-tour-nantais-2024/)|30/10/2024 au 31/10/2024|Nantes (France)|[🟢 Open until 1/7/2024](https://sessionize.com/agile-tour-nantais-2024)|
|[DevFest Strasbourg](https://devfest.gdgstrasbourg.fr/)|21/11/2024|Strasbourg (France)|[🟢 Open until 2/7/2024](https://conference-hall.io/public/event/ZZv81ZNIzVzCMBxjX1xe)|
|[DevFest Dijon](https://devfest.developers-group-dijon.fr/)|6/12/2024|Dijon (France)|[🟢 Open until 15/7/2024](https://conference-hall.io/public/event/LvvYRcF5AIhpojdn2lkQ)|
|[BDX I/O](https://bdxio.fr/)|8/11/2024|Bordeaux (France)|[🟢 Open until 19/7/2024](https://conference-hall.io/public/event/9udDKH96m45FLJmqz2Uu)|
|[PyCon.FR](https://www.pycon.fr/2024/)|31/10/2024 au 3/11/2024|Strasbourg (France)|[🟢 Open until 21/7/2024](https://cfp.pycon.fr/pyconfr-2024/cfp)|
</details>

<details>
  <summary>Raw Markdown</summary>

```
### Ongoing CFPs from 20/5/2024 to 26/5/2024

|Name|🗓️ Dates|📍 Location|🗣️ CFP Status|
|---|---|---|---|
|[We Love Speed](https://www.welovespeed.com/)|17/9/2024|Nantes (France)|[🔴 Open until 26/5/2024](https://www.welovespeed.com/2024/cfp)|
|[Agile Tour Sophia-Antipolis 2024](https://www.telecom-valley.fr/agile-tour-sophia-2024/)|26/9/2024|Biot (France)|[🔴 Open until 31/5/2024](https://sessionize.com/agile-tour-sophia-antipolis-2024/)|
|[Agile Tour Rennes 2024](https://agiletour.agilerennes.org/)|13/11/2024 au 14/11/2024|Rennes (France)|[🔴 Open until 31/5/2024](https://sessionize.com/agile-tour-rennes-2024/)|
|[DevFest Toulouse](https://devfesttoulouse.fr/)|7/11/2024|Toulouse (France)|[🟡 Open until 2/6/2024](https://sessionize.com/devfest-toulouse/)|
|[Who Run The Tech ?](https://whorunthetech.com)|28/11/2024|Rennes (France)|[🟡 Open until 4/6/2024](https://conference-hall.io/public/event/bJDribf9r5Lf2VzBgbYQ)|
|[Agile Pays Basque](https://agile-paysbasque.fr/)|6/9/2024 au 7/9/2024|Bidart (France)|[🟡 Open until 7/6/2024](https://conference-hall.io/public/event/w59YkxhTPvNuwdaRCXKc)|
|[DevFest Nantes](https://devfest.gdgnantes.com/en/)|17/10/2024 au 18/10/2024|Nantes (France)|[🟡 Open until 9/6/2024](https://conference-hall.io/public/event/AJYPylkzTDd6j3O6ADZn)|
|[Cloud Nord](https://www.cloudnord.fr/)|10/10/2024|Lille (France)|[🟡 Open until 15/6/2024](https://conference-hall.io/public/event/WhX2AfC4tuVw5Ukdd5FW)|
|[Agile en Seine 2024](https://www.agileenseine.com/)|17/9/2024 au 18/9/2024|Issy-les-Moulineaux (France)|[🟡 Open until 16/6/2024](https://sessionize.com/agile-en-seine-2024/)|
|[JUG Summer Camp](https://www.jugsummercamp.org)|6/9/2024|La Rochelle (France)|[🟡 Open until 21/6/2024](https://conference-hall.io/public/event/p1eSeAZlUj8gc8DxE8vX)|
|[Agile Tour Bordeaux 2024](https://agiletourbordeaux.fr/)|30/10/2024 au 31/10/2024|Bordeaux (France)|[🟢 Open until 30/6/2024](https://sessionize.com/agile-tour-bordeaux-2024/)|
|[Agile Tour Nantais 2024](https://agilenantes.org/agile-tour-nantais-2024/)|30/10/2024 au 31/10/2024|Nantes (France)|[🟢 Open until 1/7/2024](https://sessionize.com/agile-tour-nantais-2024)|
|[DevFest Strasbourg](https://devfest.gdgstrasbourg.fr/)|21/11/2024|Strasbourg (France)|[🟢 Open until 2/7/2024](https://conference-hall.io/public/event/ZZv81ZNIzVzCMBxjX1xe)|
|[DevFest Dijon](https://devfest.developers-group-dijon.fr/)|6/12/2024|Dijon (France)|[🟢 Open until 15/7/2024](https://conference-hall.io/public/event/LvvYRcF5AIhpojdn2lkQ)|
|[BDX I/O](https://bdxio.fr/)|8/11/2024|Bordeaux (France)|[🟢 Open until 19/7/2024](https://conference-hall.io/public/event/9udDKH96m45FLJmqz2Uu)|
|[PyCon.FR](https://www.pycon.fr/2024/)|31/10/2024 au 3/11/2024|Strasbourg (France)|[🟢 Open until 21/7/2024](https://cfp.pycon.fr/pyconfr-2024/cfp)|
```
</details>

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -5,4 +5,14 @@ This folder contains all the useful scripts.
 
 ## google-apps-script
 
-Imports the JSOn file of all the events into a Google Spreadsheet table.
\ No newline at end of file
+Imports the JSOn file of all the events into a Google Spreadsheet table.
```
Comment (scraly) [inline scripts/README.md:8]:
```suggestion
Imports the JSON file of all the events into a Google Spreadsheet table.
```

**Comment by scraly** [comment]
Hi
thanks for the PR
I think this kind of script can help other people :) 

Do you know if it's possible to add the "France" country parameter in a variable of the function?

**Review by scraly — CHANGES_REQUESTED** [review]
just a typo to fix

**Comment by thierrylee** [comment]
> Hi thanks for the PR I think this kind of script can help other people :)
> 
> Do you know if it's possible to add the "France" country parameter in a variable of the function?

Sure, of course !
Though I'll need to create a "main" function without parameters. This "main" function will called by Google App Scripts triggers which cannot provide parameters to a method call.

**Comment by scraly** [comment]
Thanks @thierrylee for the script, I think it can help people :)

**Review by scraly — APPROVED** [review]

**Pull request merged**